charts/ratify/templates/deployment.yaml

```diff
@@ -14,6 +14,9 @@ spec:
     metadata:
       labels:
         {{- include "ratify.selectorLabels" . | nindent 8 }}
+        {{- if .Values.akvCertConfig.enabled }}
+        aadpodidbinding: {{ required "aadPodIdentity must be provided when AKV cert config is enabled" .Values.akvCertConfig.aadPodIdentity  }}
```

Comment:
Since we're only adding `aadpodidbinding` label here, I'm assuming this is using the AKS pod identity add-on. For OSS, `AzureIdentity` and `AzureIdentityBinding` custom resources are required. Adding this note here so we mention using AKS pod identity add-on in the docs.

Reply:
Yes, the docs has steps to create AAD pod identity before configuring Ratify